TaskHouseUnitTests/TasksControllerUnitTests.cs

```diff
@@ -21,6 +23,19 @@ public TasksControllerUnitTests()
             controller = new TasksController(repo);
         }
 
+        private TasksController login(TasksController con)
```

Comment:
I know I'm just holding you back for nothing here, but could this one get some kind of comment explaining it, in case someone else needs to maintain it?